projects/laji/src/app/shared-modules/latest-documents/latest/short-document.component.ts

```diff
@@ -89,10 +89,8 @@ export class ShortDocumentComponent implements OnInit, OnChanges, OnDestroy {
     return getLocality$(this.translate, this.labelService, gatheringInfo, this.document);
   }
 
-  editDocument(formId, documentId) {
-    this.router.navigate(
-      this.localizeRouterService.translateRoute([this.formService.getEditUrlPath(formId, documentId)])
-    );
+  getEditDocumentRoute(formId, documentId) {
```

Comment:
this method is called on every change detection cycle and causes performance issues. This could be changed to a local variable or change this components changeDetection to onPush